README.md

```diff
@@ -554,6 +555,17 @@ The documentation for releases and `main` are available here:
 
 <br>
 
+#### Migration Guides
+
+* [1.9.0](https://pointfreeco.github.io/swift-composable-architecture/main/documentation/composablearchitecture/migratingto1.9)
```

Comment:
Sorry if I wasn't clearer, but I was wondering if we could inline these links next to the full documentation links above, instead of in its own section. GitHub isn't letting me add suggestions in the expanded file, but above it'd look something like:

```diff
   * [1.9.0](https://pointfreeco.github.io/swift-composable-architecture/1.9.0/documentation/composablearchitecture/)
+    ([migration guide](https://pointfreeco.github.io/swift-composable-architecture/main/documentation/composablearchitecture/migratingto1.9))
   <details>
     <summary>
     Other versions
     </summary>
   * [1.8.0](https://pointfreeco.github.io/swift-composable-architecture/1.8.0/documentation/composablearchitecture/)
+    ([migration guide](https://pointfreeco.github.io/swift-composable-architecture/main/documentation/composablearchitecture/migratingto1.8))
```